infra/terraform/playbooks/db_recovery.md

```diff
@@ -0,0 +1,37 @@
+# HASH DB recovery playbook
+
+Something went terribly wrong with the database, here’s how to recover.
+
+## Identify snapshot to restore
+
+The HASH database is backed up every day in a 30-minute window in the interval 03:00–11:00 UTC.
+this means that we’ll have daily, incremental backups for every day in the snapshot history of the RDS instance.
+
```

Comment:
Not sure it's worth mentioning this since it might drift from reality and isn't important to the process
```suggestion
```